06-express-and-layered-architecture/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{

```

Comment:
Please try not to include commented lines in your code. Especially not, if it's copy pasta, pleaaaaase!:)